src/app/state/coordination.js

```diff
@@ -29,31 +29,31 @@ export const DEFAULT_COORDINATION_VALUES = {
   [CoordinationType.SPATIAL_AXIS_FIXED]: false,
   [CoordinationType.SPATIAL_ROTATION_ORBIT]: 0,
   [CoordinationType.SPATIAL_ORBIT_AXIS]: 'Y',
-  [CoordinationType.SPATIAL_RASTER_LAYER]: null,
+  [CoordinationType.SPATIAL_IMAGE_LAYER]: null,
   [CoordinationType.SPATIAL_SEGMENTATION_LAYER]: null,
   [CoordinationType.SPATIAL_POINT_LAYER]: null,
   [CoordinationType.SPATIAL_NEIGHBORHOOD_LAYER]: null,
   [CoordinationType.HEATMAP_ZOOM_X]: 0,
   [CoordinationType.HEATMAP_ZOOM_Y]: 0,
   [CoordinationType.HEATMAP_TARGET_X]: 0,
   [CoordinationType.HEATMAP_TARGET_Y]: 0,
-  [CoordinationType.GENE_EXPRESSION_COLORMAP]: 'plasma',
-  [CoordinationType.GENE_EXPRESSION_COLORMAP_RANGE]: [0.0, 1.0],
-  [CoordinationType.GENE_EXPRESSION_TRANSFORM]: null,
-  [CoordinationType.GENE_FILTER]: null,
-  [CoordinationType.GENE_HIGHLIGHT]: null,
-  [CoordinationType.GENE_SELECTION]: null,
-  [CoordinationType.CELL_FILTER]: null,
-  [CoordinationType.CELL_HIGHLIGHT]: null,
-  [CoordinationType.CELL_SET_SELECTION]: null,
-  [CoordinationType.CELL_SET_HIGHLIGHT]: null,
-  [CoordinationType.CELL_SET_COLOR]: null,
-  [CoordinationType.CELL_COLOR_ENCODING]: 'cellSetSelection',
+  [CoordinationType.FEATURE_VALUE_COLORMAP]: 'plasma',
+  [CoordinationType.FEATURE_VALUE_COLORMAP_RANGE]: [0.0, 1.0],
+  [CoordinationType.FEATURE_VALUE_TRANSFORM]: null,
+  [CoordinationType.FEATURE_FILTER]: null,
+  [CoordinationType.FEATURE_HIGHLIGHT]: null,
+  [CoordinationType.FEATURE_SELECTION]: null,
+  [CoordinationType.OBS_FILTER]: null,
+  [CoordinationType.OBS_HIGHLIGHT]: null,
+  [CoordinationType.OBS_SET_SELECTION]: null,
+  [CoordinationType.OBS_SET_HIGHLIGHT]: null,
+  [CoordinationType.OBS_SET_COLOR]: null,
+  [CoordinationType.OBS_COLOR_ENCODING]: 'cellSetSelection',
```

Comment:
Not sure we should be hardcoding a value like this that is arbitrary...commented on this in #1246

Reply:
It is not arbitrary, we currently only allow `cellSetSelection` and `geneSelection` which dictate whether the points on the scatterplot are [colored](https://github.com/vitessce/vitessce/blob/b28ae1abbc5e08d296f979f0f3606c6e75365e47/src/components/interpolate-colors.js#L96) according to the currently-selected cell sets or the expression of the currently-selected gene, for example. This is why I have not changed it to `obsSetSelection` or `featureSelection` here, although we could do that in a future PR.